ush/ICpert_scripts/DTC_ensemble_config_defaults.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# This file serves as a central location for settings specific to DTC's
+# Ensemble Design project.  It is sourced by other scripts.
+#
+#-----------------------------------------------------------------------
+#
+# Flag for running the RRFS analog workflow with stochastic physics.
+#
+#RRFS_do_stoch="FALSE"
+RRFS_do_stoch="TRUE"
+#
+# Flag for running the GET_OBS_... tasks as part of the RRFS analog
+# workflow.
+#
+RRFS_do_get_obs="FALSE"
+#
+# Directory in which GEFS output from 30-hour forecasts are staged.  This
+# is output from the ensemble forecast that starts at 18Z two days prior
+# to the start day of the forecasts in the RRFS analog workflow.
+#
+GEFS_staging_dir=${GEFS_staging_dir:-"/scratch2/BMC/fv3lam/ens_design_RRFS/data/GEFS/pgrb2_combined_bn"}
+#
+# Flag for performing a test run.  Relative to the full 10 RRFS member
+# (9 GEFS member) ensemble on the 3km CONUS grid for 13 cycles with a
+# forecast length of 36 hours, this entails one or more of the following
+# changes:
+#
+# 1) using fewer members
+# 2) using a coarser grid
+# 3) running for fewer cycles
+# 4) using a shorter forecast length
+#
+do_test_run="FALSE"
+#
```

Comment:
Are all four options activated above when do_test_run="TRUE"?  If not, how can you select specific options?

Reply:
Yes, but you can also pick and choose which ones you want to override.  Note that this file contains default values (much like `config_defaults.sh` in the SRW App), and the user can include a user configure file named `DTC_ensemble_config.sh` that can override the default values.  The `README` describes this in more detail.

Reply:
Oh, whoops, I missed that this was config_defaults.sh and not config.sh.  OK, thank you!